src/fight/shooting.rs

```diff
@@ -0,0 +1,284 @@
+//! # Shooting phase Module
+//!
+//! This module contain all the functions to emulate the shooting between 2 Regiment
+//! This may evolve to multiple Regiment against multiple Regiment
+
+use crate::{
+    fight::shooting_tools, fight::AttackPosition, fight::ComputeCase, fight::FightCase,
+    fight::FightPredictionResult, regiment,
+};
+
+/// Make two units shot and return the probability that the fight occurs during the game
+///
+/// # Parameters
+/// attackers (&mut regiment::Regiment, &ComputeCase): The attacking regiment
+///
+/// defender (&mut regiment::Regiment, &ComputeCase): The defending regiment
+///
+/// # Return
+/// f64: The probability that the shoot computed occurs
+fn apply_shoot(
```

Comment:
I thought that we had separate points per phase that we would have needed to get together. I misunderstood, but now the way I understand this, there is no actual reason to keep the "shooting.rs" file because every function is related to the counting of the point in this file. It seems way better to make a new function to deal with the whole turn and the whole counting including the phase that we have when I'll deal with #45